airflow/providers/amazon/aws/triggers/ecs.py

```diff
@@ -173,6 +173,7 @@ async def run(self) -> AsyncIterator[TriggerEvent]:
                     )
                     # we reach this point only if the waiter met a success criteria
                     yield TriggerEvent({"status": "success", "task_arn": self.task_arn})
+                    return
```

Comment:
similar problem as the one discussed above here. I'm fixing it in this PR because it's tiny, but I can do a separate PR as well if you think it's better.